plugins/gitlab/src/class.ts

```diff
@@ -206,17 +209,23 @@ export class GitlabZoneApi extends GitlabApi {
     })
   }
 
-  public async getOrCreateInfraProject(zone: string) {
+  public async getOrCreateInfraProject(zone: string): Promise<ProjectSchema> {
+    return this.infraProjectsByZoneSlug.get(zone) ?? await this.createInfraProject(zone)
```

Comment:
Je propose ceci pour éviter les `undefined`
```suggestion
    if (this.infraProjectsByZoneSlug.has(zone)) {
       return  this.infraProjectsByZoneSlug.get(zone)
    }
    return await this.createInfraProject(zone)
```

Reply:
Je vois que le code à changé depuis mais juste à titre informatif :
```ts
const foo = new Map()
foo.set('bar', 'bar')

const fn1 = (key) => {
  return foo.get(key) ?? 'no value'
}

console.log(fn1('bar')) // bar
console.log(fn1('baz')) // no value


const fn2 = (key) => {
  if (foo.has(key)) {
    return foo.get(key)
  }
  return'no value'
}

console.log(fn2('bar')) // bar
console.log(fn2('baz')) // no value

```